po/pt-br.po

```diff
@@ -0,0 +1,381 @@
+# Brazilian Portuguese translation for com.vixalien.sticky.
+# Copyright (C) 2023 com.vixalien.sticky
+# This file is distributed under the same license as the sticky-notes package.
+#
+# Fúlvio Leo <fulviokaldas@outlook.com>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.vixalien.sticky\n"
+"Report-Msgid-Bugs-To: https://github.com/vixalien/sticky\n"
+"POT-Creation-Date: 2023-05-04 17:21+0200\n"
+"PO-Revision-Date: 2023-05-16 00:01-0300\n"
+"Last-Translator: Fúlvio Leo <fulviokaldas@outlook.com>\n"
+"Language-Team: Brazilian Portuguese <https://br.gnome.org/traducao >\n"
```

Comment:
Since this project is not part of the GNOME Translation Project, this translation team mail (or website) doesn't apply here. There is not actual team, so you may leave only the language name (or empty).

```suggestion
"Language-Team: Brazilian Portuguese\n"
```